.github/workflows/neon_extra_builds.yml

```diff
@@ -232,20 +232,20 @@ jobs:
 
       - name: Run cargo build
         run: |
-          mold -run cargo build --locked $CARGO_FLAGS $CARGO_FEATURES --bins --tests
+          mold -run cargo build --locked $CARGO_FLAGS $CARGO_FEATURES --bins --tests -j$(nproc)
```

Comment:
Should we add the same for our regular `.github/workflows/build_and_test.yml` workflow?

Reply:
By default, the cargo should use the maximum number of available cores. But I always prefer to specify the number of jobs. Currently I've only worked with the workflow associated with Arm64 builds, but I might check out other workflows (preferably in a separate PR).